docs/Delivery-Practices/Build-and-Release/Engineering/Quality-Standards/React-Best-Practises.md

```diff
@@ -0,0 +1,39 @@
+---
+title: React Best Practises 
+authors: 
+- Rhodri Hewitson
+---
+
+## Introduction
+
+React is a popular open-source JavaScript library used for building user interfaces. It was created by Facebook and has gained a lot of popularity among developers for its declarative and efficient approach to building web applications. TypeScript is a superset of JavaScript that adds optional static typing and other features to the language. By using TypeScript with React, developers can write more reliable, maintainable, and scalable code with fewer errors. In this context, it's important to follow best practices and guidelines to ensure that your code is of the highest quality. In this post, we've provided some guidelines and best practices for writing React code in TypeScript that can help you write better code and improve the overall quality of your application.
+
+## Guidelines for Writing React Code in TypeScript
+
+1. Use semantic HTML: Use semantic HTML tags that describe the content of the page rather than just the appearance. This will help with accessibility and search engine optimization, and make your code easier to read and maintain.
```

Comment:
It may worth adding the relevant documentation for these guidelines. Something like: "There's a lot of documentation out there for semantic HTML, an example of a popular site used for web standards is [MDN](https://developer.mozilla.org/en-US/docs/Glossary/Semantics).